unitex/src/fr/umlv/unitex/leximir/util/Utils.java

```diff
@@ -231,17 +231,20 @@ public static void InflectDelas(String lemma, String fst) throws IOException, Fi
         
         if (new File(DictionaryPath.inflectionPath + fst + ".grf").exists()) {
         	
-        	  OutputStreamWriter out= new OutputStreamWriter(new FileOutputStream(DictionaryPath.allDelas +File.separator+ "DelasTmp.dic"));
-          	new File(DictionaryPath.delafPath+File.separator+ "DelafTmp.dic").createNewFile();
-       
+        	File tmp = File.createTempFile("DELASTmp", ".dic",new File(DictionaryPath.allDelas));
+        	OutputStreamWriter out= new OutputStreamWriter(new FileOutputStream(tmp));
+        	        	
+          	File result = new File(DictionaryPath.delafPath+File.separator+ "DelafTmp.dic");
```

Comment:
`DelafTmp.dic` is still hard-coded here!

Reply:
Done.